frontend/components/site/siteBuildLogs.jsx

```diff
@@ -38,10 +38,15 @@ function getSiteBuildLogTable(buildDetails, logs, state) {
   );
 }
 
+function hasBuildTasks(buildTasks) {
```

Comment:
Can we move this into the `useBuildTasks` hook to keep all the build tasks state there?